docs/access-the-swarm/gateway.md

```diff
@@ -0,0 +1,46 @@
+---
+title: Swarm Gateways
+id: gateway
+---
+
+Swarm is a p2p network, like a Blockchain, and in order to access it, you must run a node, which will connect you directly to other user's computers across the globe.
+
+We recommend you run your own node in order to access the network. If you would like to try Swarm out, limited public gateways are provided. For advanced users, you may be interested in [running your own gateway](/docs/installation/gateways).
+
+## Public Gateways
+
+The Swarm Foundation currently provides access to these public gateways.
+
+### Swarm Foundation Gateway
+
+The [Swarm Gateway](https://gateway.ethswarm.org/) is an application that allows you to try out Swarm, you are able to upload up to 10 MB at a time for free. Upload files, directories, or even folders containing websites to host them on Swarm.
+
+You will receive a link that you can share with others who will be able to retrieve the data from any gateway, or directly from the network using their own node.
+
+`https://gateway.ethswarm.org/`
+
+### Swarm Foundation Gateway API
+
+For Dapp builders, the API Gateway provides limited access to some elements of a Bee node's API. You can use this to run your Swarm applications against, if they are able to operate within the limitations stated below. 
+
+`https://api.gateway.ethswarm.org/`
+
+### BZZ.link
+
+The [BZZ.link](https://bzz.link/) service allows you to access ENS domains with websites that are hosted on [Swarm](https://swarm.bzz.link/) such as `https://swarm.bzz.link. 
```

Comment:
I think this example is a bit confusing because of its recursive nature. Maybe I would phrase it like this

```suggestion
The [BZZ.link](https://bzz.link/) service allows you to access ENS domains and swarm references (encoded as CIDs) as subdomains under `bzz.link`, providing secure context for web applications.

If you have an ENS domain name (e.g. `example.eth`) and you set the swarm reference correctly in ENS then you may simply access your site as `example.bzz.link`. You can find more information about ENS [here](https://docs.ens.domains/).
```